storage/mysql/log_storage.go

```diff
@@ -180,6 +183,7 @@ func (m *mySQLLogStorage) hasher(treeID int64) (merkle.TreeHasher, error) {
 func (m *mySQLLogStorage) beginInternal(ctx context.Context, treeID int64) (storage.LogTreeTX, error) {
 	// TODO(codingllama): Validate treeType
 	var duplicatePolicy string
+	glog.V(4).Infof("QueryRow: %v with %v", flatten(getTreePropertiesSQL), treeID)
```

Comment:
Yeah, I thought there must be a better method. I'll just drop the logging commit.